gemini/tuning/Gemini_Supervised_Tuning_for_Q&A.ipynb

```diff
@@ -0,0 +1,923 @@
+{
```

Comment:
<p>Could you add another example to show the difference in output for the foundation model and the tuned model?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/606/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='66'/>

Reply:
<p>We're working on adding Eval. I will leave this open and add eval later. Thanks. </p>